src/VintageBookshelf.Api/Configuration/ApiConfig.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace VintageBookshelf.Api.Configuration
+{
+    public static class ApiConfig
+    {
+        public static IServiceCollection AddWebApiConfig(this IServiceCollection services)
+        {
+            services.AddControllers();
+            services.Configure<ApiBehaviorOptions>(options => options.SuppressModelStateInvalidFilter = true);
```

Comment:
What is that for?

Reply:
Model State is validated by the ASP.NET Core framework by default if not disabled. I want to have more control over the way that the Model State is validated. This means that I'll always receive incoming requests to controller's endpoints.

Reply:
So you want to check if the ModelState is valid on your code?

Reply:
Yes, I want flexibility on my end to be able to return custom error messages. For example, if we let AP.NET Core to control Model State validation, default error messages will be returned instead.

Reply:
Yopu can check this one out then: 

```
services.AddControllers().
.ConfigureApiBehaviorOptions(
setupAction => { setupAction.InvalidModelStateResponseFactory = context => { **<CUSTOM RESPONSE FACTORY>** } } );
```

Reply:
I am not sure you can use it in MVC though.